src/components/theme/DraftBackground/DraftBackground.jsx

```diff
@@ -2,15 +2,47 @@ import React from 'react';
 import { connect } from 'react-redux';
 import './draft.css';
 import { BodyClass } from '@plone/volto/helpers';
+import { withRouter } from 'react-router-dom';
+import { compose } from 'redux';
+import { runtimeConfig } from '@plone/volto/runtime_config';
 
-const DraftBackground = ({ review_state }) => {
-  const draftClass = `wf-state-${review_state}`;
-  return <BodyClass className={draftClass} />;
+/**
+ * The review_state and id don't change everytime the page is changed and because of that the draft background
+ * will apear on pages that shouldn't have it. The RAZZLE_DISABLE_DRAFT_WATERMARK varible from ENV file should have two possible values:
+ * "Hide-No-Workflow" and "Hide-All". If the variable is not present, it should follow the current logic(show the draft image everywhere),
+ * if the value is "Hide-No-Workflow", then the draft image is not shown on the /login and /controlpanel and if the value is "Hide-All", then
+ * the draft image is not visible at all.
+ * For example, if the current page is /datatable (that has the draft background) and then we go to Content Types page,
+ * the review_state and id will be the same as the ones from /datatable, so the draft background will still be present. By checking
+ * if the pathname from (from withRouter) is different than 'login' or 'controlpanel' and based on the varible from ENV,
+ * we decide if the draft backgound can be present or not.
+ * @param {Object} props
+ * @returns
+ */
+const DraftBackground = (props) => {
+  const draftClass = `wf-state-${props.review_state}`;
+  const razzleDraft =
+    runtimeConfig['RAZZLE_DISABLE_DRAFT_WATERMARK'] || 'default';
+  const isReviewableStateComponent =
```

Comment:
Maybe we can filter using `nonContentRoutes` settings? @MihaelaCretu11 could you please try? https://github.com/plone/volto/blob/1d006f84984a5706fa7926d6e9f58630b9bbfa0e/src/config/NonContentRoutes.jsx#L5
As this is a setting, it can account for more routes in future and is bulletproof.